### ISSUE TITLE
Pre-release 3.6.16779 does not show embedded files

### ISSUE THREAD

**Issue by GitHubRulesOK** [sumatrapdfreader/sumatrapdf#4563]
### Discussed in https://github.com/sumatrapdfreader/sumatrapdf/discussions/4560

Does not seem to be an issue covering this more recent change.

<div type='discussions-op-text'>

<sup>Originally posted by **lywind** September 27, 2024</sup>
In SumutraPDF 3.5.2, I can view and save embedded files in PDF
![image](https://github.com/user-attachments/assets/0c8d34cb-85c9-447c-ae49-90c370c57356)
However, in version 3.6 pre-release, the embedded files are disappeared.
![image](https://github.com/user-attachments/assets/ec1d1bf9-22b9-4bc7-87eb-76add72bdf24)
</div>

**Comment by v3nus-py** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by musikai** [comment]
it worked ok in 3.6.15987

**Comment by GitHubRulesOK** [comment]
Yes but I still wonder if there has been some change in MuPDF older Acrobat does not show the attachments as conventional in lower window but warns there is scripting at play. 
![image](https://github.com/user-attachments/assets/a4402de5-3628-42b9-addd-a68d25b9e216)
The scripts are related to showing attached files.
![image](https://github.com/user-attachments/assets/25fa098e-2972-4f56-9512-a172096cbcbf)

**Comment by musikai** [comment]
Can't speak for this file but here is mine.
[Werke für Gitarre(free).pdf](https://github.com/user-attachments/files/18426562/Werke.fur.Gitarre.free.pdf)

**Comment by GitHubRulesOK** [comment]
@musikai 
That is a better example
but odd it has a distinct path to D:\, however older SumatraPDF would see and save the file but not current pre-release
![image](https://github.com/user-attachments/assets/7df789d4-4b4c-489c-a8e6-be64af2d5c85)

**Comment by kjk** [comment]
I've opened https://bugs.ghostscript.com/show_bug.cgi?id=708587

**Comment by gawkla** [comment]
The PDF created and sent by the Polish bank mBank contains embedded files (statements). Some of these statements are embedded as HTML files. Is it possible to open them directly in the browser without saving them? Currently, I can see the embedded files, but when I click on them, nothing happens.

![Image](https://github.com/user-attachments/assets/8c514a33-81cf-4200-bc99-8fee0ed78e08)

**Comment by GitHubRulesOK** [comment]
@gawkla without a sample hard to tell is the problem you may have is the same.

Modern Global invoicing means that a PDF may often have an embedded XML invoice (etc) for machine reading, these are not "normally" intended for any user convenience, simply for commercial justification of the visible PDF surface data, in the weird hope that a PDF with embedment is "more" secure than one with a readable surface.

SumatraPDF would not be suited for such commercial extraction (and data comparison) and IMHO should not need to run such scripting for a secured VIEWER users needs.

**Comment by gawkla** [comment]
As the Polish mBank example shows, there are two types of embedded content: an HTML page inside a PDF or a PDF inside a PDF. The idea is that the main document is password-protected. I don't know why they're embedding an unencrypted PDF inside an encrypted one. For example: PDF X-change can open embedded content immediately after clicking on it. In Sumatra there is additional step: "open attachment"

![Image](https://github.com/user-attachments/assets/47a73624-e671-4bfb-83a4-0423fbcae0bf)

**Comment by GitHubRulesOK** [comment]
@gawkla 
that sounds like a slightly different but related issue,
Currently SumatraPDF FOR YOUR SAFETY (Unlike other apps) does NOT usually "open and run" internal files.
The only secure way to handle embedment's is to export (where antivirus is available) and let the user decide which workflow to use.

HOWEVER the bigger issues ARE
- Here SumatraPDF does not see the Scripted embedment's 
- Currently extraction is NYI (Not Yet Implemented)

**Comment by kjk** [comment]
@gawkla it's a different issue so please open a new bug and send me a test file (if confidential, to kkowalczyk@gmail.com see https://www.sumatrapdfreader.org/docs/How-to-submit-bug-reports)

**Comment by gawkla** [comment]
I am not sure, if it is "feature" or "issue"? I understand that currently I have to pickup option "open attachment" to open. If it is working as designed -> OK. If there is option to change this behaviour - I will open new issue to handle this.

**Comment by kjk** [comment]
Yes, working as designed.
